pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -61,6 +78,8 @@ func ConvReason(status libvirt.DomainState, reason int) api.StateChangeReason {
 		return ShutoffReasonTranslationMap[libvirt.DomainShutoffReason(reason)]
 	case libvirt.DOMAIN_CRASHED:
 		return CrashedReasonTranslationMap[libvirt.DomainCrashedReason(reason)]
+	case libvirt.DOMAIN_PAUSED:
```

Comment:
In order to test this, you can extend the tests at https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/manager_test.go#L168. They take a libvirt reason and return the translated reason.